### ISSUE TITLE
Prevent append blueimp style to head

### ISSUE THREAD

**Issue by riso348** [RobinCK/vue-gallery#75]
Hi,
is it possible to prevent append css file _blueimp-gallery/css/blueimp-gallery.min.css_ into head defined in vue-gallery.js file?
I would like to include this file custom. Im using Nuxt / Vue.
It is because google page speed has a problem with this when i include it after load.

**Comment by cizza** [comment]
I have to agree with riso348. I'm struggling with your plugin in my build as well because of the tight coupling of styles. Can't they be completely decoupled? Thank you.
